qa/workunits/rbd/cli_migration.sh

```diff
@@ -199,6 +218,47 @@ EOF
     remove_image "${dest_image}"
 }
 
+# Function to test the import of native format with namespace
+test_import_native_format_with_namespace() {
```

Comment:
Currently this function is covering "Migration from namespace to non-namespace" and "Migration from namespace to namespace" scenarios.  Let's cover "Migration from non-namespace to namespace" as well for completeness.

Reply:
covered the suggested scenario as well.